futures-util/src/compat/tokio.rs

```diff
@@ -0,0 +1,71 @@
+use crate::{future::FutureExt, try_future::TryFutureExt};
+use std::{
+    future::FutureObj,
+    task::{Executor, SpawnErrorKind, SpawnObjError},
+};
+use tokio_executor::{DefaultExecutor, Executor as _};
+
+/// An executor that delegates to `tokio`'s [`DefaultExecutor`], will panic if
+/// used in the context of a task that is not running on `tokio`'s executor.
+///
+/// # Examples
+///
+/// ```ignore
+/// #![feature(async_await, await_macro, underscore_imports, pin)]
+///
+/// use std::thread;
+/// use std::boxed::PinBox;
+///
+/// use futures::{
+///     channel::oneshot,
+///     future::FutureExt as _,
+///     future::TryFutureExt as _,
+///     executor::spawn,
```

Comment:
spawn is now macro. See my [blog post PR](https://github.com/rust-lang-nursery/futures-rs/pull/1165) for a quick intro